skipruntime-ts/core/src/skipruntime_api.ts

```diff
@@ -445,12 +448,6 @@ export type Watermarked<K extends TJSON, V extends TJSON> = {
   update?: boolean;
 };
 
-export type Notifier<K extends TJSON, V extends TJSON> = (
-  values: Entry<K, V>[],
-  watermark: string,
-  update: boolean,
-) => void;
```

Comment:
A nice thing about having this type declared is that here would be a good place to document the arguments.